### ISSUE TITLE
Ensure that links in error reports lead to efforts to handle the error

### ISSUE THREAD

**Issue by dscotese** [ACINQ/eclair#1257]
I expected that the readme file to which this error links would provide some information on how to deal with the inability of Eclair to use a wallet that contains non-segwit outputs.  That error is displayed in #1255  because I was unable to copy it.  It serves as a good example on which to base a solution in this particular case.  More generally, however, I think any link offered by a reported error (the readme file in this case) should contain something to indicate what someone can do if they found a solution to the error which they would like to be available to others.  I 

See #1255 

I have provided a fix for this in pull request #1256.  I think the error message can be improved directly (to say "To use the current bitcoin wallet, you must send any existing non-segwit UTXOs to a new `p2sh-segwit` or `bech32` address before starting Eclair.") so I might work on another pull request to do that.

**Comment by t-bast** [comment]
Closed by #1256
